src/manifolds/VectorBundle.jl

```diff
@@ -99,18 +99,22 @@ space at `p`.
 CotangentSpaceAtPoint(M::Manifold, p) = VectorSpaceAtPoint(CotangentBundleFibers(M), p)
 
 """
-    VectorBundle(M::Manifold, type::VectorSpaceType)
+    VectorBundle{TVS<:VectorSpaceType,TM<:Manifold,𝔽} <: Manifold{𝔽}
+
+Vector bundle on a [`Manifold`](@ref) `M` of type [`VectorSpaceType`](@ref).
 
-Vector bundle on manifold `M` of type `type`.
+# Constructor
+
+    VectorBundle(M::Manifold, type::VectorSpaceType)
 """
-struct VectorBundle{TVS<:VectorSpaceType,TM<:Manifold} <: Manifold
+struct VectorBundle{TVS<:VectorSpaceType,TM<:Manifold,𝔽} <: Manifold{𝔽}
```

Comment:
Should we require `𝔽` to always be the number system of `TM`?

Reply:
That would unify the parameters, yes. We also have to switch `TM´ and `𝔽` then.

Reply:
Oh but it would also mean to rework all your special types and const s that follow afterwards. Hm.

Reply:
I can switch that later.